src/Contrib/StackSdks/Masa.Contrib.StackSdks.Tsc/Tracing/ActivityExtensions.cs

```diff
@@ -93,10 +93,10 @@ private static async Task SetActivityBody(Activity activity, Stream inputSteam,
         (long length, string? body) = await inputSteam.ReadAsStringAsync(encoding);
 
         if (length <= 0)
-            return;        
+            return;
         if (length - OpenTelemetryInstrumentationOptions.MaxBodySize > 0)
         {
-            OpenTelemetryInstrumentationOptions.Logger?.LogInformation(body);
+            OpenTelemetryInstrumentationOptions.Logger?.LogInformation("Request body in base64 encode:{Body}", body);
```

Comment:
`Request body in base64 encode: {Body}`

Reply:
SetActivityBody(activity, httpRequest.Body, GetHttpRequestEncoding(httpRequest)).Wait();

Synchronous call to asynchronous? Use wait directly? on line19，line 34